Tasks/HelmInstallerV1/src/helmtoolinstaller.ts

```diff
@@ -20,10 +19,18 @@ async function configureHelm() {
 
 async function verifyHelm() {
     console.log(tl.loc("VerifyHelmInstallation"));
+    var helmVersion = await utils.getHelmVersion();
     var helmToolPath = tl.which("helm", true);
     var helmTool = tl.tool(helmToolPath);
-    helmTool.arg("init");
-    helmTool.arg("--client-only");
+    
+    // Check if using Helm 2 or Helm 3
+    if (helmVersion.startsWith("2")) {
+        helmTool.arg("init");
+        helmTool.arg("--client-only");
+    } else {
+        helmTool.arg("version");
+    }
+
     return helmTool.exec()
 }
 
```

Comment:
Bump up patch version in task.json and task.loc.json

Reply:
Done! Thanks!